glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/Azure/azure-sdk-for-go
 import:
 - package: github.com/Azure/go-autorest
-  version: ~8.4.0
+  version: ^9.1.0
```

Comment:
Let's keep this as `~9.1.0`

Reply:
Oh also, in the changelog you mention that you need 9.1.1 not 9.1.0, which is right?

Reply:
Fixed

Reply:
well, it is at least 9.1.0, will fix

Reply:
9.1.1 is for generator tests

Reply:
Fixed

Reply:
I would be interested to hear from both sides on the motivations behind the change.

Reply:
Okok, looked for the [difference](https://github.com/Masterminds/glide/blob/master/docs/versions.md)
Now that I understand it, `^` would be better I think

Reply:
Agreed, caret seems to do what I thought tilde did. :)